http/src/response/future.rs

```diff
@@ -122,14 +113,7 @@ impl InFlight {
                     source: Some(Box::new(source)),
                 }))
             }
-            Poll::Pending => {
-                return InnerPoll::Pending(ResponseFutureStage::InFlight(Self {
-                    future: self.future,
-                    guild_id: self.guild_id,
-                    invalid_token: self.invalid_token,
-                    tx: self.tx,
-                }))
-            }
+            Poll::Pending => return InnerPoll::Pending(ResponseFutureStage::InFlight(self)),
```

Comment:
Similar to above, why was this change made?

Reply:
Cleanup / much easier to read